etcd-client/src/etcd_client_simple.cpp

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Eliminar forward declarations problemáticas
```

Cppcheck (2.19.0)

[error] The code contains unhandled character(s) (character code=195). Neither unicode nor extended ascii is supported.

(unhandledChar)